elastic/logs/challenges/logging-querying.json

```diff
@@ -22,14 +22,14 @@
         "clients": {{ p_bulk_indexing_clients }},
         "ignore-response-error-level": "{{error_level | default('non-fatal')}}"
       },
+      {
+        "name": "compression-stats",
+        "operation": {
+          "operation-type": "compression-statistics",
+          "param-source": "create-datastream-source"
+        }
+      },
     {% endif %}
-    {
```

Comment:
by moving this inside the conditional, we are sure it won't break https://github.com/elastic/elasticsearch-sizing/blob/master/logging/run_books/index_compression.md, right?

Reply:
LGTM on the first pass. I'd want a test to ensure this concern is a non-issue.

Reply:
This change does not break that use case because index compression stats rely on indexing. I'll look into adding a test.

Reply:
I actually have no idea how to test that, this is another reason to prefer having logic outside of the tracks.

Reply:
Did a little bit of digging, I think the index compression methodology won't get broken with this change because it uses a different challenge: https://github.com/elastic/rally-tracks/blob/7354a0307ae67463adabdde4fc1719f132f2ce80/elastic/logs/challenges/logging-disk-usage.json#L92